csharp/Spotflow.EventHubConsumer/Program.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+
+using System.Collections.Concurrent;
+using System.Text;
+
+var storageConnectionString = "<< CONNECTION STRING FOR THE STORAGE ACCOUNT >>";
+var blobContainerName = "<< NAME OF THE BLOB CONTAINER >>";
+
+var eventHubsConnectionString = "<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>";
+var eventHubName = "<< NAME OF THE EVENT HUB >>";
+var consumerGroup = "<< NAME OF THE EVENT HUB CONSUMER GROUP >>";
+
+var storageClient = new BlobContainerClient(
+    storageConnectionString,
+    blobContainerName);
+
+var processor = new EventProcessorClient(
+    storageClient,
+    consumerGroup,
+    eventHubsConnectionString,
+    eventHubName);
+
+var partitionEventCount = new ConcurrentDictionary<string, int>();
+
+async Task processEventHandler(ProcessEventArgs args)
```

Comment:
I would move the methods' declaration to the end (convention).

Reply:
Moved